docs/components/search-bar/index.html

```diff
@@ -0,0 +1,31 @@
+--- 
+  title: Menus 
+  also: 
+    elements/hx-menu: <hx-menu>
+    elements/hx-menuitem-group: <hx-menuitem-group>
+    elements/hx-menuitem: <hx-menuitem>
+--- 
+{% extends 'component.njk' %} 
+{% block content %}
+  <section id="vue-searchBarDemo">
+    <h2 class="hxSectionTitle" id="search-bar">Basic Search Bar</h2>
+    <p>
+      State:
```

Comment:
Please move demo configuration to a column, right of the demo (for docs consistency).

See Box, Panels, Popovers, Status Pills, or Tooltips for examples.

Reply:
👍